### PR TITLE
Controller and tests for filter queries

### PR DESCRIPTION
**Pull request by CatoLeanTruetschel**
See #107 

There is still an issue that the query service cannot handle exact numeric filtering. This is tracked by #155 and will be fixed in a follow up.

### CODE REVIEW

File: backend/src/filter/EntityFilterConditionExecutor.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  OfTypeCondition,
+} from '../shared/queries';
+import FilterConditionVisitor from './FilterConditionVisitor';
+
+export class EdgeFilterConditionExecutor extends FilterConditionVisitor {
+  private edges: Edge[] = [];
+
+  public filterEdges(edges: Edge[], filter: FilterCondition): EdgeDescriptor[] {
+    this.edges = edges;
+    this.visit(filter);
```
Comment (PianoRollRepresentation) [inline backend/src/filter/EntityFilterConditionExecutor.ts:16]: Currently, the check whether required properties are set or the filtering, is not implemented, right? So `visit()` does not do anything right now, correct?

Comment (PianoRollRepresentation) [reply backend/src/filter/EntityFilterConditionExecutor.ts:16]: Also, I don't understand how `visit()` will filter the edges without getting them as a parameter.

**Review by PianoRollRepresentation — COMMENTED** [review]
There is one point I don't really understand. But maybe I am just getting smth wrong there...

**Review by PianoRollRepresentation — APPROVED** [review]

File: backend/src/filter/EntityFilterConditionExecutor.ts (hunk 1 above)
Comment (CatoLeanTruetschel) [reply backend/src/filter/EntityFilterConditionExecutor.ts:16]: Of course `visit`is fully implemented. It is an instance method and has full access to `this.edges` that is set in line 15.

**Pull request merged**